src/HTSPTypes.h

```diff
@@ -151,17 +152,41 @@ struct SRecording
     priority  = 50; // kodi default - "normal"
   }
   
+  bool IsStopTimeInThePast () const
+  {
+    time_t now = time(NULL);
+    return (now > static_cast<time_t>(stop + (stopExtra * 60)));
+  }
+
   bool IsRecording () const
   {
-    return state == PVR_TIMER_STATE_COMPLETED ||
-           state == PVR_TIMER_STATE_ABORTED   ||
-           state == PVR_TIMER_STATE_RECORDING;
+    if (state != PVR_TIMER_STATE_ERROR)
+    {
+      // determine type by state.
+      return state == PVR_TIMER_STATE_COMPLETED ||
+             state == PVR_TIMER_STATE_ABORTED   ||
+             state == PVR_TIMER_STATE_RECORDING;
+    }
+    else
+    {
+      // determine type by time (both timers and recordings may have errors).
+      return IsStopTimeInThePast();
```

Comment:
Should be 'IsStartTimeInThePast()', otherwise running recordings will not show up in the "recordings" window.